client/blocks/reader-full-post/_content.scss

```diff
@@ -248,28 +248,17 @@
 		margin: auto;
 	}
 
-	.wp-block-buttons {
```

Comment:
Why were these classes removed? (They aren't the correct styles, admittedly, but I feel like we still need them, without knowing a better place to put them)